sdk/attestation/azure-security-attestation/src/main/java/com/azure/security/attestation/AttestationAsyncClient.java

```diff
@@ -117,7 +123,8 @@
      * @return attestation response for Trusted Platform Module (TPM) attestation.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<TpmAttestationResponse> attestTpm(TpmAttestationRequest request) {
-        return this.serviceClient.attestTpmAsync(request);
+    public Mono<String> attestTpm(String request) {
+        return this.serviceClient.attestTpmAsync(new com.azure.security.attestation.implementation.models.TpmAttestationRequest().setData(request.getBytes(StandardCharsets.UTF_8)))
+            .map(response -> new String(response.getData(), StandardCharsets.UTF_8));
```

Comment:
What happens in the error scenario? You may want to handle errors as well.

Reply:
I have no idea - this is a skinny wrapper around the generated code with the bare minimum needed to preserve compatibility. I'll be adding support for properly written Track2 implementations in a future pull request.